UrbanAirship.iOS.Basement.nuspec

```diff
@@ -15,7 +15,7 @@
 
    <files>
      <file src="src/AirshipBindings.iOS.Basement/bin/release/AirshipBindings.iOS.Basement.dll" target="lib/Xamarin.iOS10" />
-     <file src="src/AirshipBindings.iOS.Basement/bin/Release/AirshipBindings.iOS.Basement.resources/**" target="content/AirshipBindings.iOS.Basement.resources" />
+     <file src="src/AirshipBindings.iOS.Basement/bin/Release/AirshipBindings.iOS.Basement.resources/**" target="content/Basement.resources" />
```

Comment:
do we risk having conflicts with other libs? should we add a `ua` prefix?

Reply:
I am not sure but we can add the prefix to avoid any conflit just in case.

Reply:
Probably a good idea. "Core" or "Basement" could be anything.